### PR TITLE
642 (part 2) update large files for a part of bundles

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes # .

### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/mednist_reg/configs/metadata.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.0.4",
+    "version": "0.0.5",
     "changelog": {
```
Comment (KumoLiu) [inline models/mednist_reg/configs/metadata.json:4]: Do we need update the monai version for this bundle?

**Review by KumoLiu — APPROVED** [review]
Thanks for the update, overall looks good to me.

File: models/mednist_reg/configs/metadata.json (hunk 1 above)
Comment (yiheng-wang-nv) [reply models/mednist_reg/configs/metadata.json:4]: I think we can keep the old version. The reason why I update for some other bundles is that our CI uses python 3.10 and cannot support old torch versions.

Comment (KumoLiu) [reply models/mednist_reg/configs/metadata.json:4]: Ok, but I guess the ci have already test with the latest monai version, so the bundle can work with the new version, right?

Comment (yiheng-wang-nv) [reply models/mednist_reg/configs/metadata.json:4]: for each bundle, CI uses it's corresponding monai/torch/... versions.

**Pull request merged**